reference/build_helpers/cmake.rst

```diff
@@ -133,6 +133,23 @@ An example of the value of this property could be:
 
     --config Release
 
+parallel
+++++++++
+
+**Defaulted to**: ``True``
+
+Run CMake process in parallel for compilation, installation and testing. This attribute gets the value for the class constructor and
```

Comment:
should we add some comment how [CONAN_CPU_COUNT](https://docs.conan.io/en/latest/reference/env_vars.html#conan-cpu-count) can affect this option?

Reply:
Already referenced in `tools.cpu_count()`